easycv/datasets/detection/raw.py

```diff
@@ -114,15 +115,15 @@ def visualize(self, results, vis_num=10, score_thr=0.3, **kwargs):
                     detection_classes.append(None)
                 else:
                     detection_classes.append(
-                        np.array([class_names[id] for id in classes_id]))
-            results['detection_classes'] = detection_classes
+                        np.array([class_names[int(id)] for id in classes_id]))
+        else:
+            detection_classes = results.get('detection_classes', [])
```

Comment:
add some comment when we need to convert class_id to class name, and when not

Reply:
done!